applications/adv_networking_bench/cpp/testing/benchmark_utils.py

```
@@ -486,6 +488,76 @@ def parse_gpunetio_benchmark_results(log: str) -> BenchmarkResults:
+        r"lost:\s+(\d+) \| exceed MD:\s+(\d+) \| bad RTP hdr:\s+(\d+) \|\s+(\d+)%"
+    )
+    benchmark_summary_rtp_receiver = (
+        r"\[stream_index\s+(\d+)\] Got\s+(\d+) packets \|\s+(\d+\.\d+) GB \|\s+\|\s+"
+        r"consumed:\s+(\d+) \| unconsumed:\s+(\d+) \|\s+"
+        r"lost:\s+(\d+) \| bad RTP hdr:\s+(\d+) \|"
+    )
+
+    matches = re.findall(benchmark_summary_ipo_receiver, log)
+    if matches:
+        (stream_index, total_rx_packets, throughput, dropped, consumed,
+         unconsumed, lost, exceed_md, bad_rtp_hdr, extra_info) = matches[0]
```

Ruff (0.14.2)

Unpacked variable `extra_info` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)